thirdeye/thirdeye-pinot/src/test/java/com/linkedin/thirdeye/tools/RunAdhocDatabaseQueriesTool.java

```diff
@@ -180,7 +90,60 @@ public static void main(String[] args) throws Exception {
       System.exit(1);
     }
     RunAdhocDatabaseQueriesTool dq = new RunAdhocDatabaseQueriesTool(persistenceFile);
-
+    List<Map<String, List<String>>> targetLevelList = new ArrayList<>();
+    List<List<Map<String, String>>> overridePropertyList = new ArrayList<>();
+
+
+//    List<Integer> idList = Arrays
+//        .asList(954637, 954638, 954639, 954640, 954641, 954642, 954643, 954644, 954645, 954646,
+//            954647, 954648);
+//    int idIdx = 0;
+
+    // Override config for 12/21 - 12/27
+    // Override Config for SU_Request Desktop
+    targetLevelList.add(getTargetLevelSet("SU_Request_from_Desktop"));
```

Comment:
Sure. This is a good suggestion. I intend to use the 'GenericDatabaseAccessTool.java' only for initialize the series of 'dao' for database interaction. Not the adhoc io method.  We'd better put the specific method in individual onboarding tools, like the 'Clone' and 'Regnerate'. Will clean this up as suggested.